modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -68,7 +68,11 @@ object types {
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
     rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
-    oneTimeRewards: List[OneTimeReward] = List.empty
+    oneTimeRewards: List[OneTimeReward] = List(
+      // Transferring final balance of 4,343,029,488,479,231 from DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS
```

Comment:
I would update this comment to also include a comment of the rounded amount so it's easier for someone to read (the exact amount should be present but also the rounded float amount.)

Reply:
Actually, considering prior approvals, maybe just add this to the PR comment

Reply:
actually it should be in the commit description rather than the pull request as it won't be stored in the git repository